src/Common/tests/System.Net/Sockets/Performance/SocketPerformanceTests.cs

```diff
@@ -8,7 +8,7 @@
 using Xunit;
```

Comment:
The SocketPerformanceTest changes shouldnt be in here. I'm going to put those tests in a separate PR shortly.